swaggerDocs.js

```diff
@@ -0,0 +1,27 @@
+const swaggerUI = require('swagger-ui-express');
+const swaggerJSDoc = require('swagger-jsdoc');
+
+const options = {
+  definition: {
+    openapi: '3.0.0', // Specification (optional, defaults to swagger: '2.0')
+    info: {
+      title: 'Community Group API', // Title (required)
+      version: '1.0.0', // Version (required)
+    },
+    // basePath: '/',
+  },
+  // Path to the API docs
+  apis: ['./src/routes/groups/groups.router.js'],
+};
+
+// Initialize swagger-jsdoc -> returns validated swagger spec in json format
+const swaggerSpec = swaggerJSDoc(options);
+
+module.exports = (app) => {
+  app.use('/api-docs', swaggerUI.serve, swaggerUI.setup(swaggerSpec));
+}
```

Comment:
If I understand this correctly, this will add an endpoint that serves the swagger docs programmatically.  That's a good idea, but please render the swagger spec out and check it into the repo so I can see how it looks.